skins/10h16/js/lib/view_handler/warning_box.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+var objectAssign = require( 'object-assign' ),
+	_ = require( 'underscore' ),
+
+	WarningBox = {
+		issueBox: null,
+		validationFunction: null,
+		update: function ( fieldValue ) {
+			if ( this.validationFunction( fieldValue ) ) {
+				this.issueBox.show();
+			}
+			else {
+				this.issueBox.hide();
+			}
+		}
+	},
+
+	createHandler = function ( issueBoxElement, shouldShowWarningFunction ) {
```

Comment:
Although the other view handles also don't have doc comments, I'd wish we had them here.